db/seeds/sandbox_data.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "faker"
+
+Faker::Config.locale = "en"
+
+logger = Logger.new($stdout)
+logger.formatter = proc do |_severity, _datetime, _progname, msg|
+  "#{msg}\n"
+end
+
+cohort = Cohort.find_or_create_by!(start_year: "2021")
+delivery_partner = DeliveryPartner.find_or_create_by!(name: Faker::Company.name)
+
+providers = ["Capita", "Teach First", "UCL", "Best Practice Network", "Ambition", "Education Development Trust"]
```

Comment:
Pretty sure now that we are loading `initial_seed` we will want to use the lead providers it makes, rather than making new ones. Or you will have 12 LPs I guess :)

Reply:
I think it'll find them and use them, I'll double check though.